open-apparel-registry/open-apparel-registry
# Add django-simple-history

## Overview

- add django-simple-history library
- create HistoricalFacility audit model
- display history log for Facility on Django admin

Connects #519 

## Demo

![Screen Shot 2019-05-16 at 11 02 05 AM](https://user-images.githubusercontent.com/4165523/57864749-75810680-77ca-11e9-958f-f7e569b5acdc.png)
![Screen Shot 2019-05-16 at 11 02 54 AM](https://user-images.githubusercontent.com/4165523/57864750-75810680-77ca-11e9-94a7-c35233a0fd80.png)
![Screen Shot 2019-05-16 at 11 02 30 AM](https://user-images.githubusercontent.com/4165523/57864752-75810680-77ca-11e9-8508-d77023439fe9.png)

## Testing Instructions

On this branch
- `./scripts/update`
- `./scripts/manage resetdb`
- `./scripts/manage processfixtures`
- `./scripts/manage createsuperuser`
- `./scripts/server`
- sign into the django admin
- visit a Facility object and verify that you can see its history
- update a facility, then verify that you can see and revert the change
- use `./scripts/manage shell_plus` to interact with a HistoricalFacility object

## Checklist
